spring-kafka/src/main/java/org/springframework/kafka/config/AbstractKafkaListenerContainerFactory.java

```diff
@@ -60,6 +65,14 @@
 
 	private DeDuplicationStrategy<K, V> deDuplicationStrategy;
 
+	private Boolean pauseEnabled;
+
+	private Long pauseAfter;
```

Comment:
I'd say that something like `pauseAfter > 0` is like and indicator from the top that `pause` is `Enabled`.
Just looks strange to require to configure both options.
Although we may configure it to `true`, when there is an explicit call to `pauseAfter`.
WDYT?

BTW is there some validation that `pauseAfter > 0`?
Or don't we need it ?